docs/source/install/index.rst

```diff
@@ -54,6 +54,7 @@ Once it completes successfully, you will see the following output:
     Ubuntu 14.04 / 16.04 <deb>
     RHEL 7 / CentOS 7 <rhel7>
     RHEL 6 / CentOS 6 <rhel6>
+    Ansible Playbooks <ansible>
```

Comment:
Right now, there's not really enough content here to justify having a dedicated Ansible page. Either we put this content in another page (e.g. refer to it from the main install page), or we overhaul the older doc, push Ansible as the 'preferred/supported' way of doing it, with notes that older Puppet/Chef stuff exists